_health-care/_js/_form.js

```diff
@@ -18,17 +18,60 @@ var HealthApp = (function() {
     var dateString = "";
 
     if (date.getMonth() < 10) {
-      dateString += "0"
+      dateString += "0";
     }
     dateString += (date.getMonth() + 1) + "/";
 
     if (date.getDate() < 10) {
-      dateString += "0"
+      dateString += "0";
     }
     dateString += date.getDate() + "/";
 
     return dateString + date.getFullYear();
   }
+  function extractMonetaryValueTransform(formData, arg) {
+    return Number(extractTransform(formData, arg)).toFixed(2).toString();
+  }
+  function validatePresence(input) {
+    var valid = true;
```

Comment:
Normally I like single exit points in functions, but for this situation, I think having a `valid` variable actually makes the code harder to grok cause you have to verify that no one has an conditional logic based on valid earlier.  I would rewrite this as

```
function validatePresence(input) {
if (input.value === '') {
  return false;
}

return true;
}
```

In this case, the clear path is you only get a true if you hit the bottom. Everything else is effectively an exception that terminates the rest of the function processing.